teleport/assets/logs/teleport.yaml

```diff
@@ -164,6 +164,7 @@ pipeline:
 
           parse_audit_session_data %{_log_prefix}\s+session.data\s+(addr.local:%{ipOrHost:network.client.ip}:%{port:network.client.port}\s+)?(addr.remote:%{ipOrHost:network.destination.ip}:%{port:network.destination.port})?\s+cluster_name:%{notSpace:teleport.cluster_name} code:%{notSpace:teleport.code} ei:%{notSpace:teleport.eid} event:%{notSpace:teleport.event_type} login:%{notSpace:teleport.login} namespace:%{notSpace:teleport.namespace} private_key_policy:%{notSpace:teleport.private_key_policy} rx:%{notSpace:teleport.bytes_read} server_hostname:%{notSpace:network.host.name} server_id:%{notSpace:teleport.server_id} sid:%{notSpace:teleport.sid} time:%{notSpace:teleport.time} tx:%{notSpace:network.bytes_written} uid:%{notSpace:teleport.uid} user:%{notSpace:teleport.user} user_kind:%{notSpace:teleport.user_kind}\s+%{regex("[^:]*"):log.file}:%{number:log.line_number}.*
 
+          parse_failed_user_login %{_log_prefix}\s+user.login\s+(addr.local:%{ipOrHost:network.client.ip}:%{port:network.client.port}\s+)?(addr.remote:%{ipOrHost:network.destination.ip}:%{port:network.destination.port})?\s+cluster_name:%{notSpace:teleport.cluster_name} code:%{notSpace:teleport.code} ei:%{notSpace:teleport.eid} error:%{regex("[\\[]*.*\\]"):teleport.event_error}\s+event:%{notSpace:teleport.event_type}\s+method:%{notSpace:teleport.method}\s+success:%{notSpace:teleport.success}\s+time:%{notSpace:teleport.time}\s+uid:%{notSpace:teleport.uid}\s+user:%{notSpace:teleport.user}\s+user_agent:%{regex("[a-zA-Z/0-9.(;_),]+"):http.useragent}\s%{notSpace}
```

Comment:
`%{_log_prefix}\s+user.login\s+(addr.local:%{ipOrHost:network.client.ip}:%{port:network.client.port}\s+)?(addr.remote:%{ipOrHost:network.destination.ip}:%{port:network.destination.port})?\s+cluster_name:%{notSpace:teleport.cluster_name}` appears in `parse_failed_user_login` and `parse_audit_session_start`. Can we refactor that into a support rule?

Also it seems that even more of the rule would match except that `parse_audit_session_start` uses `\s` and `parse_failed_user_login` uses spaces (eg `\s+code:%{notSpace:teleport.code}\s+ei` vs ` code:%{notSpace:teleport.code} ei`). Is there a need for those to be different? If not I'd recommend we standardize and move as much of this to a support rule as possible.

Reply:
Done!